src/lib/multiledger.js

```diff
@@ -41,7 +41,11 @@ Multiledger.prototype.getType = function (ledgerId) {
 }
 
 Multiledger.prototype.getStatus = function () {
-  return _.every(this.ledgers, (ledger) => ledger.isConnected())
+  return _.every(this.ledgers, (ledger) => ledger.isConnected()) ? {
```

Comment:
Couldn't this be also written:
```js
  return { 
    ledgersStatus: _.every(this.ledgers, (ledger) => ledger.isConnected()) ? healthStatus.statusOk : healthStatus.statusNotOk
  }
```
Sounds more natural not to duplicate the object maybe?